scripts/publish.sh

```diff
@@ -18,22 +15,26 @@ usage() {
 
 error() {
     echo "error: $1"
-    exit ${2:-1}
+    exit "${2:-1}"
 }
 
 confirm()  {
     echo -n "$1 [yN]: "
-    read confirm
+    read -r confirm
     [[ "$confirm" != "y" ]] && [[ "$confirm" != "Y" ]] && error "user aborted"
 }
 
+package_json() {
+    node -e "console.log(require('./package.json').$1)"
```

Comment:
it feels dirty

Reply:
Yeah, but at least we know `node` will be available. One less step.